x/oracle/keeper/recent_params.go

```diff
@@ -0,0 +1,76 @@
+package keeper
```

Comment:
When is the data used? Can you explain briefly?

Reply:
This is like the recent_msgs, which is used to refill the cache:
the aggregation is affected by 3 elements:
1. validatorset(: `votingPower`)
2. params(: `tokenFeeder`)
3. messages (: including price_with_`votingPower` for `tokenFeeder`)
So when we do the refill, we just use these 3 data to reproduce the aggregation process.
But i think the recent_params could be optimized with carefully check like validator_update_block so that we don't need to keep the history params, currently this is a straightforward way. I will update in a new PR